.github/deploy/destroy.ps1

```diff
@@ -29,6 +29,18 @@ Write-Host "Initialize deployment" -ForegroundColor Green
 Write-Host "  Now Destroying Parent Resource Group!" -ForegroundColor Red
 
 az group delete --name $resourceGroupName --yes --no-wait
-
 Write-Host "  Parent Resource Group Deleted" -ForegroundColor Green
 
+Start-Sleep -Seconds 30
+
+# ATM, deleting the resource group above does not completely delete the managed resource group.
+# Thus, we delete it here. If this is fixed by Azure, we can remove this step.
+if (az group exists --name "$($resourceGroupName)Cluster" = true ) {
+  Write-Host "  Managed resource group exists, Deleting it..." -ForegroundColor Green
+  az group delete --name "$($resourceGroupName)Cluster" --yes --no-wait
```

Comment:
I have tested it out in this PR: https://github.com/spetlr-org/spetlr/pull/150 . It seems that the resource group somehow is protected by systems (SPNs) to delete it......

Reply:
oh. I forgot to checl open PRs :) . Thanks. I abandon this PR then.